website/components/Head/index.js

```diff
@@ -0,0 +1,33 @@
+import NextHead from 'next/head';
+import PropTypes from 'prop-types';
+
+const Head = ({ meta }) => {
+  return (
+    <NextHead>
+      <title>{meta?.title}</title>
+      <meta content={meta?.description} name="description" />
+      <meta property="og:type" content={meta?.type} />
+      <meta property="og:site_name" content="Strapi Design System" />
+      <meta property="og:description" content={meta?.description} />
+      <meta property="og:title" content={meta?.title} />
+    </NextHead>
+  );
+};
+
+Head.propTypes = {
+  meta: PropTypes.shape({
```

Comment:
why to pass the meta as an object and not directly title / description and type as props?

Reply:
This one too